### ISSUE TITLE
Unexpected  when seeking value

### ISSUE THREAD

**Issue by seirl** [jgeboski/bitlbee-facebook#13]
This happens all the time.

facebook - Error: Parser: 1:0: Unexpected  when seeking value

**Comment by Zopieux** [comment]
Did this started when you added someone to your friends (or some friend added you)?

**Comment by dequis** [comment]
Never seen this one, still getting it? How frequent is it? ("all the time" isn't very precise)

**Comment by toogy** [comment]
Just got it too:
```
10:51:01  @root | facebook - Error: Parser: 1:0: Unexpected  when seeking value
```
(first time ever though)

**Comment by dequis** [comment]
Submitted #19 which should give more helpful error messages to figure out wtf this is

**Comment by seirl** [comment]
22:41 <seirl> JSON len=365: \026FD8yuKOn3byECEc+tzB98j
22:41 <dx> brutal     
22:41 <dx> jgeboski: does this look like thrift ^

**Comment by seirl** [comment]
JSON len=455: \026Bojn2mQRmMFCf5j8ODBOWE
JSON len=384: \026ByNhgnbVUAFEV1OjZL3hyO
JSON len=386: \026DRFfVwl9OAqGd0L3vJaCyN
JSON len=364: \026E4TBUAgOF2+HG9ncUAmz8g

**Comment by jgeboski** [comment]
Since I have not been able to recreate this, could someone grab a stack trace from when `g_set_error()` is called within `fb_json_new()`?

**Comment by jgeboski** [comment]
This has been resolved as of 75b3f5b.
